ng build --prod fails when HttpInterceptorModule imported in app.module

Just by adding the following statement on my app.module file, production build with AOT fails.

``` ts
import { HttpInterceptorModule } from 'ng-http-interceptor';
```
running production  build, I am getting following error. 

```ts
ERROR in D:/Projects/Git-Repos/ui/src/$$_gendir/node_modules/
ng-http-interceptor/dist/ng-http-interceptor.ngfactory.ts (10,26): 
Cannot find module 'httpInterceptor'.
```

I have following in my dependencies
"ng-http-interceptor": "^3.1.0",
 "proxy-polyfill": "^0.1.7",

and following in devdependencies
"tslib": "1.6.0",

Can you please help me know, what may I doing wrong?

--- a/tsconfig.es2015.json
+++ b/tsconfig.es2015.json
@@ -22,7 +22,7 @@
     "strictMetadataEmit": true,
     "skipTemplateCodegen": true,
     "flatModuleOutFile": "ng-http-interceptor.js",
-    "flatModuleId": "httpInterceptor",
+    "flatModuleId": "ng-http-interceptor",
     "annotateForClosureCompiler": true
   }
 }

